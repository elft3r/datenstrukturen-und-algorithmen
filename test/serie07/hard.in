10
26 125
326 560 161 477 705 600 5 190 661 857 70 205 421 940 125 283 270 252 472 456 176 603 78 827 815 267
2 20 1 18 1 20 3 13 13 13 1 11 6 4 9 12 15 1 19 7 7 9 4 16 8 18 
21 138
110 102 389 437 457 603 938 561 384 428 609 447 830 479 375 499 666 163 593 968 205
13 11 2 2 10 1 12 20 19 14 12 1 14 16 2 1 20 3 17 20 6 
23 131
849 933 488 239 972 493 440 872 888 670 152 604 229 749 872 699 901 52 493 780 9 381 588
3 5 5 2 2 16 4 17 20 19 18 4 8 11 4 7 2 14 4 5 20 7 4 
25 145
821 618 294 372 715 220 602 341 258 601 840 35 863 630 417 223 686 751 115 401 706 437 486 595 137
3 8 4 12 9 6 5 5 17 13 5 1 3 2 11 13 7 8 3 12 18 13 13 19 10 
21 127
152 463 633 943 65 566 923 435 18 909 837 519 965 127 872 154 950 923 744 280 693
17 5 5 7 12 4 20 11 6 17 5 12 16 16 6 17 19 11 14 11 6 
26 125
585 385 133 937 725 346 488 389 218 494 620 842 565 181 760 183 274 297 903 151 213 514 410 718 919 212
8 1 15 3 6 13 4 12 1 1 3 16 20 19 4 10 9 3 20 5 20 5 3 15 1 14 
26 117
631 288 346 557 975 323 740 931 877 869 533 149 699 509 784 974 357 44 457 979 601 234 291 61 401 732
11 7 8 5 20 11 20 6 20 19 2 19 14 13 4 11 12 9 16 3 4 11 9 2 17 5 
23 156
586 770 377 735 939 660 277 254 558 791 359 926 731 511 308 680 722 74 73 651 16 925 368
18 8 3 20 20 4 8 4 15 15 6 9 19 4 17 13 12 15 13 5 13 4 9 
25 118
739 881 838 654 553 292 819 696 580 571 871 438 146 697 506 354 410 544 781 407 161 166 700 493 356
15 7 6 5 7 8 2 19 18 2 8 2 20 11 20 9 14 10 8 6 9 17 2 12 18 
24 110
578 837 177 352 807 602 677 770 704 178 297 192 345 901 825 448 589 188 583 14 692 328 897 101
5 17 14 1 20 15 7 15 3 5 18 11 17 17 10 18 9 13 7 5 8 9 17 13 
