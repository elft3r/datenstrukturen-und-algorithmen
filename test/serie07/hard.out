8262 1 2 3 5 6 9 10 13 14 18 20 22 23 24 25
8607 3 4 5 6 7 10 11 12 13 15 16 17 18 19 20 21
12470 1 2 3 4 5 6 7 8 9 12 13 14 15 16 17 19 20 22 23
10429 1 2 3 5 6 7 8 10 11 13 14 15 17 18 20 21 22 23
10017 2 3 4 6 10 11 12 13 15 17 18 19 21
10701 1 2 4 5 6 7 8 9 10 11 12 15 18 19 20 22 23 24 25
9774 1 4 5 8 10 11 13 14 15 16 20 21 26
10399 2 3 4 5 6 8 10 11 12 13 14 16 17 20 22 23
10481 1 2 3 4 5 6 7 8 10 11 12 14 18 19 20 23
7963 1 2 4 7 9 14 15 17 19 21 22 23
